.github/workflows/nix_tests.yaml

```diff
@@ -38,4 +38,4 @@ jobs:
       env:
         HF_TOKEN: ${{ secrets.HF_TOKEN }}
     - name: Rust tests.
-      run: nix develop .#test --command cargo test
+      run: nix build .#checks.x86_64-linux.rust -L
```

Comment:
Why does this contains the platform and I don't see it in the nix definition ?
Can't we make this platform agnostic?

Reply:
In the flake itself it's platform agnostic, but the issues is that `nix flake check` currently [doesn't support running a single check](https://github.com/NixOS/nix/issues/8881). Otherwise we could use `nix flake check .#rust`. So this is a hack to run only that test.

I'll update this PR to use `nix eval --impure --raw --expr 'builtins.currentSystem'`.